src/otx/core/model/entity/base.py

```diff
@@ -106,3 +109,62 @@ def map_class_names(src_classes: list[str], dst_classes: list[str]) -> list[int]
             else:
                 src2dst.append(-1)
         return src2dst
+
+
+class OVModel(OTXModel, Generic[T_OTXBatchDataEntity, T_OTXBatchPredEntity]):
+    """Base class for the OpenVINO model."""
+
+    def __init__(self, config: DictConfig) -> None:
+        self.model_name = config.pop("model_name")
+        self.model_type = config.pop("model_type")
+        self.async_inference = config.pop("async_inference", False)
+        self.num_requests = config.pop("max_num_requests", get_default_async_reqs_num())
+        self.use_tp_mode = config.pop("use_tp_mode", False)
+        self.config = config
+        super().__init__()
+
+    def _create_model(self) -> nn.Module:
+        """Create a OV model with help of Model API."""
+        from openvino.model_api.adapters import OpenvinoAdapter, create_core, get_user_config
+        from openvino.model_api.models import Model
+
+        plugin_config = get_user_config("AUTO", str(self.num_requests), "AUTO")
+        if self.use_tp_mode:
+            plugin_config["PERFORMANCE_HINT"] = "THROUGHPUT"
+
+        model_adapter = OpenvinoAdapter(
+            create_core(),
+            self.model_name,
+            max_num_requests=self.num_requests,
+            plugin_config=plugin_config,
+        )
+
+        return Model.create_model(model_adapter, model_type=self.model_type)
+
+    def _customize_inputs(self, entity: T_OTXBatchDataEntity) -> dict[str, Any]:
+        # restore original numpy image
+        images = [np.transpose(im.numpy(), (1, 2, 0)) for im in entity.images]
```

Comment:
So, it means RGB -> BGR? Do OV_API needs the BGR images?

Reply:
It's not about changing color channel. It is about changing an order of tensor dimensions,
`C x H x W => H x W x C`.

Reply:
Correct, to infer OV model, we need to run np.arrays with H x W x C dimension. 
I checked that BGR is by default used in our OTX. Answering to your question, OV requires BGR images, not RGB

Reply:
Previously, we were using bgr-rgb swap in OTX 1.x, since the original torch models are all trained in RGB, and model API doesn't color channels order.
Does OTX 2.0 train models in BRG?